src/Extension/FluentSiteTreeExtension.php

```diff
@@ -155,7 +155,7 @@ public function updateCMSFields(FieldList $fields)
         if (!FluentState::singleton()->getLocale()) {
             return;
         }
-
+        parent::updateCMSFields($fields);
```

Comment:
To some extent this makes the API a little less predictable, because the parent's method will not always be run. Maybe we should move the `FluentState::singleton()->getLocale` check into the parent class and just call `parent::updateCMSFields($fields)` in this class - what do you think?

Reply:
Scratch that - you'd need to return something more explicit from the parent method and check for it here. Just leave it =)